readme.md

```diff
@@ -0,0 +1,42 @@
+# Sample files
+
+These samples all use Juypter Notebooks.
+
+Python requirements:
+```
+from mindee import Client
+import json
+import cv2
+from matplotlib import pyplot as plt
+import numpy as np
+import math
+from IPython.lib.pretty import pretty
```

Comment:
Some polishing suggestions:
```
pip install -r requirements.txt
```

and you create a `requirements.txt` file to place all the dependencies

Python developers / enthusiasts are used to have a requirements.txt file to install deps, with the version specifiers cf. https://github.com/mindee/doctr/blob/main/requirements.txt

And here it's shown how to import, but not how to install those (a package can be indexed on pypi with a different name than the name under which you import it: for instance `python-doctr` on pypi, but `import doctr`

Reply:
in that regards, "json" and "math" are part of the standard library, so you don't need to add them as deps!